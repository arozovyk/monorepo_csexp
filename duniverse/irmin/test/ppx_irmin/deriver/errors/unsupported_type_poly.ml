type t = { v : 'a. 'a } [@@deriving irmin]
