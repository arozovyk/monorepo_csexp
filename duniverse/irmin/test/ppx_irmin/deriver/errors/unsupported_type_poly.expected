File "unsupported_type_poly.ml", line 1, characters 0-42:
Error: ppx_irmin: universally-quantified type 'a . 'a encountered. Irmin types must be grounded.
